templates/field/field--multiple-values--comma.html.twig

```diff
@@ -0,0 +1,9 @@
+{#
+/**
+ * @file
+ * Field is used for multiple values field to set comma separator between rows.
+ *
+ * @see ./core/themes/stable/templates/field/field.html.twig
+ */
+#}
+{% for item in items %}{{ item.content }}{% if not loop.last %}, {% endif %}{% endfor %}
```

Comment:
If we create a template for each separator then we could update all the other templates that are doing exactly this.
For e.g.:
field--node--oe-event-languages--oe-event--full.html.twig

For other templates the separator is "|" so we could add another template just for separating values of multivalue fields. Otherwise, just use this code inside the template file where it's included so we don't have an additional template.